corehq/apps/app_manager/views/apps.py

```diff
@@ -585,7 +585,9 @@ def _version_text(result):
     if request.method == "POST":
         clear_app_cache(request, domain)
         from_app_id = request.POST.get('from_app_id')
-        app_copy = import_app_util(from_app_id, domain)
+        app_copy = import_app_util(from_app_id, domain, {
+            'created_from_template': from_app_id,
```

Comment:
We've used this to track various iterations of template apps in the past. We currently use it [here](https://github.com/dimagi/commcare-hq/blob/5e625e67ec2fc41dc3ee2267d1eed01c2613eee5/corehq/apps/app_manager/views/apps.py#L510) to identify the apps created by the appcues new user workflow.